code-challenges.js

```diff
@@ -14,14 +14,38 @@ var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
-
-
+//make a function that takes a number parameter
+const boilingTemperature = (num) => {
+    //determine if num is below , at, or above boiling point with condtionals (boiling point is 212 F)
+    if (num <= 211) {
+        //output `${num} is below bp, at, or above.    
+        console.log(`${num} is below boiling point`);
+    } else if (num === 212) {
+        console.log(`${num} is at boiling point`);
+    } else if (num >= 213) {
+        console.log(`${num} is above boiling point`);
+    } else {
+        return "Please enter a number"; //edges in case a number is not entered (for practice)
+    }
+}
+//test
+console.log(boilingTemperature(temp3));
 // --------------------2) Create a function that takes in two arrays of numbers and returns a single array with all the values sorted least to greatest.
 // Use the test variables provided below. Expected output: [-9, -7, 0, 3, 7, 8, 9, 13, 22, 36]
 
 var myNumbers1 = [3, 7, 0, 36, -9]
 var myNumbers2 = [8, -7, 22, 9, 13]
+var myNumbers3 = []
+//make a function that takes two arrays of numbers
+const arraySorter = () => {
+    myNumbers3 = myNumbers1.concat(myNumbers2).sort((a, b) => a - b) //the sort function by default organizes elements alphabetically so it has to be given more specific instructions to sort numerically. The added instructions tell sort to sort the numbers ascending from least to greatest, to make it descend we could change "a-b" to "b-a". I got the solution from the syllabus but could use help understanding what exactly the a, b represent in this context I understand how it functions but not necessarily what each part represents fully. 
```

Comment:
no worries, a and b are placeholders for the two values at different indexs that sort is looking at. Sort is a relly sophisticated algorithm that compares values and moves them around the array. It requires a fair amount of processing power with large arrays.